iceoryx_examples/iceoptions/iox_subscriber_with_options.cpp

```diff
@@ -41,8 +41,14 @@ int main()
     // publisher has send. The history request ensures that we at least get the last 5
     // samples sent by the publisher when we subscribe (if at least 5 were already sent
     // and the publisher has history enabled).
+
+    // we do not require the publisher to support the history we request,
+    // i.e. we will still connect even if its historyCapacity is smaller than what we request
+
     //! [history]
     subscriberOptions.historyRequest = 5U;
+
+    subscriberOptions.requiresPublisherHistorySupport = false;
     //! [history]
```

Comment:
Could you either run geoffrey or update the example readme manually? It might also ne a good idea to mention the new option in more detail in the readme, similar to the comment you added here

Reply:
Was already done in a previous commit.

Reply:
yeah, just overlooked it